jarbas/settings.py

```diff
@@ -69,7 +69,7 @@
     'debug_toolbar.middleware.DebugToolbarMiddleware',
 ]
 
-if not DEBUG:
+if ENVIRONMENT == 'production':
```

Comment:
Actually we don't need that. If I'm not wring, WhiteNoise auto-disables itself if `DEBUG == True`, so we need to simply re-add the `DEBUG` setting.

Reply:
Are you sure about this? At second sight, the middleware does not fully disable itself for this reason. Sources [[1]](https://github.com/evansd/whitenoise/blob/master/whitenoise/middleware.py) and [[2]](https://github.com/evansd/whitenoise/blob/master/docs/django.rst).